content/docs/introduction/technical-preview-free-tier.md

```diff
@@ -7,44 +7,22 @@ redirectFrom:
 
 The Neon cloud service is available for free during the Technical Preview. For information about how the Technical Preview fits within Neon's release plans, please refer to the [Neon roadmap](/docs/reference/roadmap).
 
-Technical Preview Free Tier users can create up to three projects in Neon. Projects created using the Technical Preview Free Tier are subject to the following limits:
+The Technical Preview Free Tier has the following limits:
 
-- Project data size is limited to 10GB
-- The Point in Time Reset (PITR) window is limited to 7 days of _reasonable usage_
-- The compute node is limited to 1 vCPU and 256MB of RAM
+- 1 Neon project
+- 10 branches
+- 3 GB of data per branch
+- 3 compute nodes (endpoints)
+- 1 vCPU with a limit of 4GB of RAM per compute node (with 512MB of RAM allocated to PostgreSQL shared buffers)
+- A point-in-time recovery (PITR) window of 7 days of _reasonable usage_
 
 <Admonition type="note">
 Neon intends to offer a Free Tier beyond the Technical Preview period. The limits associated with that tier will be defined in the coming months. Technical Preview Free Tier limits are subject to change over the course of the Technical Preview.
 </Admonition>
 
 ## Data Size
 
-Neon separates storage and compute and stores data in its own internal format.
-The 10GB data size limit applies to the logical size of a Neon project. The logical size is the sum of all relation sizes in the project.
-
-To check the logical size of the databases in your Neon project, run the following query:
-
-```sql
-SELECT pg_size_pretty(sum(pg_database_size(datname)))
-FROM pg_database;
-```
-
```

Comment:
This command works well for me on our compute, maybe we can keep it?

Reply:
@ololobus Does this query make sense now that we have moved our data size limits to branches (3GB per branch)? I am assuming this query gives you a project-wide sum.

Reply:
> I am assuming this query gives you a project-wide sum.

No, it gives somewhat the logical branch (timeline) size, IIUC. Actually, I think that it's impossible to give the actual project (tenant) size from the running compute. This info is only available on the pageserver

cc @lubennikovaav @LizardWizzard please, correct me if I'm wrong

Reply:
OK. I have added it back to the docs, but I have moved it to this branches page, which will be published soon with https://github.com/neondatabase/website/pull/336.
https://websitemain62807-dpricemanageusers.gatsbyjs.io/docs/manage/branches/#check-the-data-size

Reply:
> No, it gives somewhat the logical branch (timeline) size, IIUC. Actually, I think that it's impossible to give the actual project (tenant) size from the running compute. This info is only available on the pageserver

Sounds correct